tests/test_vmtkScripts/conftest.py

```diff
@@ -14,25 +14,25 @@
 ##       Richard Izzo (Github @rlizzo)
 ##       University at Buffalo
 
-from __future__ import print_function
+from __future__ import print_function, absolute_import
```

Comment:
not that we need this since we're not supporting 2.x
maybe we could open a dedicated issue on this, so that we remove all future imports across the repo

Reply:
TODO